src/main/resources/db/migration/V31__regelverk.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE regelverk(
```

Comment:
@madsop-nav Trenger vi en tabell for regelverk hvor verdi aldri endrer? Det kan være en enum i kodebase. Kan vi ikke inkludere i Behandling tabell med alle eksisterende verdi som NASJONAL(hvis EØS for tilbakekreving ikke startet enda). Ifølge implementasjon skal vi oppdatere regelverk for alle nye behandlinger, men ikke for eksisterende behandlinger. Da kan vi sette regelverk column til nullbar også.

Reply:
Det vil nok fungere å ha det som ein rein varchar i databasen, men eg liker å ha typesikkerheit der eg kan. Da er vi heilt sikre på at ingen klarer å lagre noko anna enn akkurat nasjonal og EØS. Det ligg som ein enum i kodebasen også, men eg synes at dei to heller utfyller enn erstattar kvarandre.

Reply:
@madsop-nav Eg er vel einig med Saikat her. Tror Nav har gått meir og meir bort frå å ha kodeverk i db, men heller bruke Enums. Og iallefall team-familie har det slik.